### PR TITLE
Update the Photon ID SFs for UL17

### PR DESCRIPTION
**Pull request by rchatter**
PR created on behalf of Prasant.
Adds the recently derived UL17 Pho. ID SFs

### CODE REVIEW

File: Systematics/python/flashggDiPhotonSystematics2017_Legacy_cfi.py [hunk 1]
```diff
@@ -274,14 +275,15 @@
...
-        cms.PSet( lowBounds = cms.vdouble( 0.0, 0.85 ) , upBounds = cms.vdouble( 1.5, 999.0 ) , values = cms.vdouble( 1.0003 ) , uncertainties = cms.vdouble( 0.0000 )  ) ,
-        cms.PSet( lowBounds = cms.vdouble( 1.5, 0.0  ) , upBounds = cms.vdouble( 6.0, 0.9   ) , values = cms.vdouble( 1.0003 ) , uncertainties = cms.vdouble( 0.0000 )  ) ,
-        cms.PSet( lowBounds = cms.vdouble( 1.5, 0.9  ) , upBounds = cms.vdouble( 6.0, 999.0 ) , values = cms.vdouble( 1.0004 ) , uncertainties = cms.vdouble( 0.0000 ) ) ) ) 
+        cms.PSet( lowBounds = cms.vdouble( 0.0, 0.0  ) , upBounds = cms.vdouble( 1.5, 0.85  ) , values = cms.vdouble( 1.0021 ) , uncertainties = cms.vdouble( 0.0014 )  ) ,
+        cms.PSet( lowBounds = cms.vdouble( 0.0, 0.85 ) , upBounds = cms.vdouble( 1.5, 999.0 ) , values = cms.vdouble( 1.0001 ) , uncertainties = cms.vdouble( 0.0038 )  ) ,
+        cms.PSet( lowBounds = cms.vdouble( 1.5, 0.0  ) , upBounds = cms.vdouble( 6.0, 0.9   ) , values = cms.vdouble( 1.0061 ) , uncertainties = cms.vdouble( 0.0014 )  ) ,
```
Comment (youyingli) [inline Systematics/python/flashggDiPhotonSystematics2017_Legacy_cfi.py:285]: Hi @rchatter , the uncertainty 0.0014 in this line is different from the slide which is 0.0013. Is this an accuracy issue?

**Review by youyingli — CHANGES_REQUESTED** [review]

File: Systematics/python/flashggDiPhotonSystematics2017_Legacy_cfi.py (hunk 1 above)
Comment (rchatter) [reply Systematics/python/flashggDiPhotonSystematics2017_Legacy_cfi.py:285]: @prasantrout could you please take a look at the comment above?

Comment (Prasant1993) [reply Systematics/python/flashggDiPhotonSystematics2017_Legacy_cfi.py:285]: Hii,
  
   Yes, actually the systematic quoted in the file is the quadrature sum of statistical uncertainty and the total systematic uncertaity on the scale factor.  Statistical uncertainty is generally very small compared to the systematic uncertainty.  For that low R9 bin in the endcap,  the systematic is found to be,  S =  sqrt(0.0004**2 + 0.0014**2) = 0.0014.    For all the other (eta, R9) bins, it is the same as the total systematic uncertainty in the scale factor.

Comment (Prasant1993) [reply Systematics/python/flashggDiPhotonSystematics2017_Legacy_cfi.py:285]: Sorry, My mistake. S = sqrt(S1*S2 + S2*S2), S1 = 0.0004 and S2 = 0.0014

**Review by youyingli — APPROVED** [review]

File: Systematics/python/flashggDiPhotonSystematics2017_Legacy_cfi.py (hunk 1 above)
Comment (Prasant1993) [reply Systematics/python/flashggDiPhotonSystematics2017_Legacy_cfi.py:285]: S1 = 0.0004, S2 = 0.0013, S = 0.0014

**Comment by youyingli** [comment]
Hi @rchatter , could you please revert two commits (19d66bd, 6cc65e9) in your branch "UL17IDSF_Update" about  UL2018 update to avoid the mixture? And then I will merge this PR, thanks!

**Comment by rchatter** [comment]
Hi You-Ying,

I messed up the branch in trying to revert and added more commits than needed to fix by hand.
But the branch now has only Prasant's file update. Let me know if you see something else in out of place.

Thanks,
Rajdeep

**Comment by youyingli** [comment]
Thanks! Merged!

**Pull request merged**